config/development.env

```diff
@@ -0,0 +1,13 @@
+NODE_ENV=production
```

Comment:
Is NODE_ENV=production right?

Reply:
Yes. All deployed code in any environment should operate as NODE_ENV=production. The only place that sometimes changes is when running tests (sometimes convenient to set NODE_ENV=test) or when running locally. NODE_ENV has little to do with deployment environment, which confusingly uses similar terms.